handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -65,8 +65,13 @@ public static SslContextBuilder forServer(File keyCertChainFile, File keyFile) {
     /**
      * Creates a builder for new server-side {@link SslContext}.
      *
-     * @param keyCertChainInputStream an input stream for an X.509 certificate chain in PEM format
-     * @param keyInputStream an input stream for a PKCS#8 private key in PEM format
+     * @param keyCertChainInputStream   an input stream for an X.509 certificate chain in PEM format. The caller is
+     *                                  responsible for calling {@link InputStream#close()} once {@link #build()} was
```

Comment:
I think it should be 'is' instead of 'was'.

Reply:
hmm.. no native speaker here. I thought "was called" is correct. But no idea :D

Reply:
Lol same here, I can understand XD.

Reply:
Change `whom` to `who` in the title. 

:p

Reply:
or `after build() has been called.`